composer.json

```diff
@@ -7,6 +7,7 @@
         "ext-iconv": "*",
         "sensio/framework-extra-bundle": "^5.2",
         "sensiolabs/security-checker": "^4.1",
+        "symfony/asset": "*",
```

Comment:
It would be better to setup a precise version of the package like:
```json
    "symfony/asset": "^4.1",
```
Same for all the dependencies with `"*"`